assets/stylesheets/navigation.scss

```diff
@@ -3,6 +3,7 @@
     border: none;
     -webkit-box-shadow: 0 1px 2px rgba(0, 0, 0, 0.3);
     box-shadow: 0 1px 2px rgba(0, 0, 0, 0.3);
+    background-color: #ffffff;
```

Comment:
Is this necessary for the actual change or an oversight?

Reply:
Yes, it's necessary. Without it the navbar is transparent.

Reply:
I see, thx.

@Martchus I think it's ok that we define this color here unless you think there is a better way

Reply:
I fixed it to use the sass variable, but: peeking around the other CSS stuff there seems to be lot in plain CSS color codes. That's something to be fixed in the medium run, I guess. Or at least to be discussed.

Reply:
I know that there are a lof of raw colors which should be improved. Thanks for making it not worse. It would be nice if 'Use sass variable' would be squashed into 'Make the navbar non-transparent'.